### PR TITLE
117801355 Product display css error

### PR DESCRIPTION
**Pull request by temi-adediran**
#### What does this PR do?

Properly displays products on all pages.
#### Description of Task to be completed?

It removes unwanted spaces between product images and displays them four per row on all category pages. It also modifies the products pagination to display 16products per page. 
#### How should this be manually tested?

When users click on a category link, each product image will be displayed four per row.
#### What are the relevant pivotal tracker stories?
#117801355

### CODE REVIEW

File: app/views/landing/_product.html.erb [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+  <%= link_to product_path(product) do %>	  
+   	<div class="card"> 
+  	  <div class="card-image">
+        <%= image_tag product.product_image_links.first.link_name %>
+      </div>
+      <div class="card-content center">
+        <span><%= product.name %></span><br>
+        <span class="price">&#x20A6;<%= product.price %></span>
+      </div>
+  	</div>
```
Comment (makinwab) [inline app/views/landing/_product.html.erb]: **Nitpick - Code Quality**
Please adjust the formatting/indentation in this file.

**Comment by makinwab** [comment]
Nice work @andela-tfowotade. Can we make the number of products per page 32 as against 30? This is so that we have 4 products as the last set of products displayed on a page and not 2.

**Comment by makinwab** [comment]
There also exists a `fix(test)` commit message that needs adjustment.

File: app/views/landing/_product.html.erb (hunk 1 above)
Comment (temi-adediran) [inline app/views/landing/_product.html.erb]: It looks properly indented on my system, I'm surprised it's showing this way here. I'll correct it.

**Comment by temi-adediran** [comment]
Good morning, thanks. I'll correct that now.

On Wed, Jun 15, 2016 at 8:01 AM, Makinwa Olubukola <notifications@github.com

> wrote:
> 
> Nice work @andela-tfowotade https://github.com/andela-tfowotade. Can we
> make the number of products per page 32 as against 30? This is so that we
> have 4 products as the last set of products displayed on a page and not 2.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/andela/getmyshop/pull/103#issuecomment-226104754, or mute
> the thread
> https://github.com/notifications/unsubscribe/ANhKwLqSCljlj_iJiGyNSBEw4nZ0HGN6ks5qL6NdgaJpZM4I1tJa
> .

## 

_Temitope Fowotade /_\* Andela.com
http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature *
_Software Developer / 08021170082 / @Andela http://bit.ly/andelatwitter_

*"The startup that's harder to get into than Harvard" - *CNN
http://snip.ly/HfId
See what The Wall Street Journal http://snip.ly/u9AI has to say.

**Comment by temi-adediran** [comment]
Those were commits from my previous branch. Don't know why it's showing
under this branch. I'll work on correcting that.

Thanks.

On Wed, Jun 15, 2016 at 8:02 AM, Makinwa Olubukola <notifications@github.com

> wrote:
> 
> There also exists a fix(test) commit message that needs adjustment.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/andela/getmyshop/pull/103#issuecomment-226104901, or mute
> the thread
> https://github.com/notifications/unsubscribe/ANhKwI9QWWsUFtHzXEJuWJEGdYv2S0kbks5qL6OVgaJpZM4I1tJa
> .

## 

_Temitope Fowotade /_\* Andela.com
http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature *
_Software Developer / 08021170082 / @Andela http://bit.ly/andelatwitter_

*"The startup that's harder to get into than Harvard" - *CNN
http://snip.ly/HfId
See what The Wall Street Journal http://snip.ly/u9AI has to say.

File: app/views/products/_related_products.html.erb [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+          <div class="card card-no-shadow">
+            <div class="card-image">
+              <span><%= image_tag related_product.product_image_links.first.link_name %></span>
+            </div>
+            <div class="card-content center">
+              <span><%= related_product.name %></span><br>
+              <span class="price">&#x20A6;<%= related_product.price %></span>
+            </div>
+          </div>
+        <% end %>
```
Comment (andela-anwocha) [inline app/views/products/_related_products.html.erb]: :+1: Great work on the proper use of materialize in placing the `card-content` element as opposed to what was done previously.

Another possible way to have iterated collection `@related_products` is:
`<%= render partial: 'products/partial_name' collection: @related_products as: 'related_product' %>`

Then place html code in `products/partial_name`

**Pull request merged**